src/tridactyl.d.ts

```diff
@@ -213,6 +213,10 @@ declare function html(
     ...values: any[]
 ): HTMLElement
 
+declare namespace browser.search {
```

Comment:
I removed this too eagerly in my earlier PR

Reply:
Make it a separate one, there's a free t-shirt in it, remember! :)